static/js/util/util.js

```diff
@@ -359,3 +359,15 @@ export function validateYear(string) {
   }
   return year;
 }
+
+
+export function makeProfileImageUrl(profile) {
+  let imageUrl = `${SETTINGS.edx_base_url}/static/images/profiles/default_120.png`.
+  //replacing multiple "/" with a single forward slash, excluding the ones following the colon
+    replace(/([^:]\/)\/+/g, "$1");
+  if (profile.profile_url_large) {
+    imageUrl = profile.profile_url_large;
+  }
+
+  return imageUrl;
+}
```

Comment:
I think that will remove the character before the double slash as well. I'm not a regex expert either, I just moved the above code from the existing Dashboard component